rom/dev/src/main.rs

```diff
@@ -187,7 +188,11 @@ pub extern "C" fn rom_entry() -> ! {
         CfiCounter::corrupt();
     }
 
-    #[cfg(not(feature = "no-fmc"))]
+    // FIPS test hooks mode does not allow handoff to RT to prevent incorrect/accidental usage
```

Comment:
NIT: Should be "handoff to FMC"

Reply:
Good point, that is more accurate. This is changed now